examples/upload-file-via-browser/.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+const config = {
+  extends: [
+    'standard',
+    'standard-react'
+  ],
+  parser: 'babel-eslint',
+  plugins: [
+    'promise'
+  ],
+  rules: {
+    'jsx-quotes': [2, 'prefer-double'],
+    'linebreak-style': [2, 'unix'],
+    'no-nested-ternary': 2,
+    'no-unused-vars': 2,
+    'no-var': 2,
+    'promise/always-return': 1,
+    'promise/catch-or-return': 1,
+    'promise/no-native': 2
+  }
+}
+
+module.exports = config
```

Comment:
well, we need a bit of lint config so that aegir doesn't get upset about react/jsx stuff.  I simplified by replacing this one with the .eslintrc from `examples/bundle-webpack`.